docs/team/kwanhw.md

```diff
@@ -20,15 +20,17 @@ Given below are my contributions to the project.
   * Justification: This improves the usability of the application, as it provides more options for the user to search for contacts.
 
 * **Code contributed**:
-  * 2000+ LoC contributed to the project
+  * 2800+ LoC contributed to the project
```

Comment:
Add a link to reposense, it is a requirement according to the PPP page.

Reply:
Added in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/322/commits/9acfd793b792ca9c3e94edf65a30df349281b70d